spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'user_show', type: :feature do
```

Comment:
- [ ] Kindly check on your test. I run into the error below while running your tests.

![image](https://user-images.githubusercontent.com/61408860/192567955-9de888f0-06ef-461e-b06d-40046c8beaef.png)

- [ ] Kindy ensure all the tests are passing.